src/docs/patterns/steppers.mdx

```diff
@@ -100,14 +100,17 @@ If certain dimensions are not specified, refer to the dimensions suggested in th
 
 Use the following components to implement this pattern:
 
-**Angular** and **Ionic** - Angular Material:
+**Angular** and **Ionic**:
 
--   [Stepper](https://material.angular.io/components/stepper/overview)
+-   Angular Material:
+    -   [Stepper](https://material.angular.io/components/stepper/overview)
+-   @pxblue/angular-components
+    -   [Mobile Stepper](https://pxblue-components.github.io/angular/?path=/info/components-mobile-stepper--readme)
 
 **React** - Material UI:
 
 -   [Stepper](https://material-ui.com/components/steppers/)
 
 **React Native** - @pxblue/react-native-components:
 
--   Pager (coming soon)
+-   Stepper (coming soon)
```

Comment:
Should we break these out (by library) like above in the angular section and other pages?